x-pack/plugin/sql/qa/src/main/resources/agg.sql-spec

```diff
@@ -563,6 +567,8 @@ SELECT MIN(salary) min, MAX(salary) max, gender g, languages l, COUNT(*) c FROM
 // group by with literal
 implicitGroupByWithLiteral
 SELECT 10, MAX("salary") FROM test_emp;
+literalWithGroupBy
```

Comment:
Please have the following tests:
- `literalWithGroupBy` (withoutAlias)
- `literalsWithGroupBy` (withoutAlias)
- `aliasedLiteralWithGroupBy` (can have mixed aliased and unaliased)
- `aliasedLiteralsWtihGroupBy` (can have mixed aliased and unaliased)
-  `literalsWithMultipleGroupBy` (aliased and unaliased literals, with more than one column in the `GROUP BY` which can also be a mix of aliased/unaliased)